src/main/webapp/app/services/visualizations/stackedverticalbar-chart.service.js

```diff
@@ -44,6 +44,7 @@
                     result['axisScaleLabel'] = VisualizationUtils.getPropertyValue(record.properties, 'Axis Scale Label');
                     result['showLegend'] = VisualizationUtils.getPropertyValue(record.properties, 'Show Legend');
                     result['legendPosition'] = VisualizationUtils.getPropertyValue(record.properties, 'Legend position').toLowerCase();
+                    result['dateFormate'] = VisualizationUtils.getPropertyValue(record.properties, 'Datetime format');
```

Comment:
This text used 4 times.. Create a const

Reply:
for every configuration, we are directly using the name of the property. 
we are planning to create common servicer to process metadata 
that we can use in flair-bi and flair-notification
at that time we can create  const file for property name 
leave it for now